sda/cmd/ingest/ingest.go

```diff
@@ -322,22 +322,22 @@ func (app *Ingest) ingestFile(correlationID string, message schema.IngestionTrig
 		}
 	case "":
 		// Catch all for inboxes that doesn't update the DB
-		log.Warnln("ingesting unregisterd file")
+		log.Infof("ingesting unregistered file, correlation-id: %s", correlationID)
```

Comment:
```suggestion
		log.Infof("registering file, correlation-id: %s", correlationID)
```

Maybe this is more logical?

Reply:
Not really since the ingestion of the actual file will begin here. Keep in mind that this case will only be triggered if someone runs this stack with third party inbox i.e. not the `s3Inbox` or the `sftpInbox`.

Reply:
Now I think your @aaperis suggestion makes more sense, since this case is only valid for those not using s3inbox and for them, it is a normal process of `ingest` to register the file and obtain the fileID. 

Apart from that, I also updated the next log message slightly: from `InsertFile` to `register file`. 
Fixed in the commit 03bd7310